static/js/invite.js

```diff
@@ -115,11 +117,13 @@ function generate_multiuse_invite() {
         beforeSend,
         success(data) {
             ui_report.success(
-                i18n.t('Invitation link: <a href="__link__">__link__</a>', {
+                i18n.t('Invitation link: <a href="__link__" id="link">__link__</a>', {
```

Comment:
This ID is super non-specific; pick something like `multiuse_invite_link`.

Reply:
okok sure, will make that change